lambdas/process-cri-callback/src/main/java/uk/gov/di/ipv/core/processcricallback/service/CriCheckingService.java

```diff
@@ -231,8 +231,10 @@ public JourneyResponse checkVcResponse(
             return JOURNEY_VCS_NOT_CORRELATED;
         }
 
-        if (!vcs.stream().allMatch(VcHelper::isSuccessfulVc)) {
-            return JOURNEY_FAIL_WITH_NO_CI;
+        for (var vc : vcs) {
```

Comment:
`isSuccessfulVc` now throws a checked exception, so we can't easily use it in a stream

Reply:
![](https://media.giphy.com/media/l41lFU69ZXu03qWrK/giphy.gif?cid=790b7611lko3cxs40a0h1vfoyrd1laefktdpzavjwbz90ran&ep=v1_gifs_search&rid=giphy.gif&ct=g)